HRTool/src/main/java/app/controllers/EmployeeRestController.java

```diff
@@ -1,22 +1,213 @@
 
 package app.controllers;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.servlet.ServletContext;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.rest.core.annotation.RepositoryEventHandler;
+import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
+import app.generator.CVGenerator;
 import app.model.Employee;
+import app.model.Project;
+import app.model.ProjectInfo;
+import app.model.TagCloud;
+import app.model.TagCloudEnum;
 import app.repository.EmployeeRepository;
+import app.repository.TagCloudRepository;
+
+import com.lowagie.text.DocumentException;
 
 
 @RestController
-@RequestMapping("/employees")
+@RequestMapping("/resources/employees")
 @RepositoryEventHandler(Employee.class)
 public class EmployeeRestController {
 	
+	@Autowired
+	EmployeeRepository employeeRepository;
+	
+	@Autowired
+    private ServletContext servletContext;
+	
+	@Autowired
+	TagCloudRepository tagCloudRepository;
+	
+
+	/**
+	 * 
+	 * @return all {Employee}s from repository
+	 */
+	
+	@RequestMapping(method = RequestMethod.GET)
+	public ResponseEntity get(){
+		
+		return new ResponseEntity(employeeRepository.findAll(), HttpStatus.OK);
+	}
+	
+	
+	/**
+	 * 
+	 * @param emp - {Employee} for save
+	 * @return {ResponseEntity} with {HttpStatus.OK}
+	 */
+	@RequestMapping(method = RequestMethod.PUT)
+	public ResponseEntity put(@RequestBody Employee emp){
+		
+		employeeRepository.save(emp);
+		return new ResponseEntity(HttpStatus.OK);
+	}
+	
+	/**
+	 * 
+	 * @param id - {Employee} id for deletion
+	 * @return {HttpStatus.NOT_FOUND} or {HttpStatus.OK}
+	 */
+	@RequestMapping(value = "/{id}" , method = RequestMethod.DELETE)
+	public ResponseEntity delete(@PathVariable("id") int id){

```

Comment:
This method should be provided by default.
http://docs.spring.io/spring-data/rest/docs/current/reference/html/#events